ci/images/debian/Dockerfile.dotnet8

```diff
@@ -94,7 +94,11 @@ RUN set -e; \
         ;; \
     esac; \
     dotnet workload list \
- && nuget help
+ && nuget help \
+ && if [ -n "$NUGET_REPO" ]; then \
+      dotnet nuget remove source "nexus"; \
+      dotnet nuget add source "https://api.nuget.org/v3/index.json" --name "nuget.org"; \
+    fi
```

Comment:
It get's removed here

Reply:
Ideally there will be an rm command or invocation to remove towards the end?

Reply:
That is exactly what is happening here -- I remove the configuration I set and restore the old one